test/python/circuit/library/test_mcmt.py

```diff
@@ -285,6 +285,19 @@ def test_gate_with_parameters_vchain(self):
         self.assertEqual(circuit.num_parameters, 1)
         self.assertIs(circuit.parameters[0], theta)
 
+    def test_mcmt_circuit_as_gate(self):
+        """Test the MCMT plugin is only triggered for the gate, not the same-named circuit.
+
+        Regression test of #13563.
+        """
+        circuit = QuantumCircuit(2)
+        gate = RYGate(0.1)
+        mcmt = MCMT(gate=gate, num_ctrl_qubits=1, num_target_qubits=1)
+        circuit.append(mcmt, circuit.qubits)  # append the MCMT circuit as gate called "MCMT"
+
+        transpiled = transpile(circuit, basis_gates=["u", "cx"])
+        self.assertTrue(Operator(transpiled).equiv(gate.control(1)))
+
```

Comment:
Just one super-tiny question: why do we check for equivalence rather than equality?

Reply:
Changed to equality in 0d26bc0e6 🙂